src/mudlet.pro

```diff
@@ -111,19 +111,23 @@ VERSION = 4.17.2
 
 # if you are distributing modified code, it would be useful if you
 # put something distinguishing into the MUDLET_VERSION_BUILD environment
-# variable to make identification of the used version simple
-# the qmake BUILD variable is NOT built-in
-BUILD_TEST = $$(MUDLET_VERSION_BUILD)
-isEmpty( BUILD_TEST ) {
+# variable (it should use '-' as the first character) to make identification of
+# the used version simple the qmake BUILD variable is NOT built-in
+BUILD = $$(MUDLET_VERSION_BUILD)
```

Comment:
Further down the file this variable `BUILD` was being checked - but because it had gotten renamed by me (a long time back - to `BUILD_TEST` here) this messed things up. Renaming this is I think the key to fixing the problem - but why it had not shown up before #6822 went in is not clear to me yet.